VideoRenderer/VideoRenderer/VideoStreamViewController.swift

```diff
@@ -74,7 +74,7 @@ public final class VideoStreamViewController: UIViewController, RendererProtocol
         provider: { VideoStreamViewController() }
     )
     
-    private var observer: PlayerObserver?
+    private var observer: SystemPlayerObserver?
     private var pictureInPictureObserver: PictureInPictureControllerObserver?
     
     private var timeObserver: Any?
```

Comment:
No reason putting different observing in one place. This will limit us in changing them.